odxtools/mux.py

```diff
@@ -0,0 +1,243 @@
+# SPDX-License-Identifier: MIT
+# Copyright (c) 2022 MBition GmbH
+
+from dataclasses import dataclass
+from typing import Optional, Dict, Any, List
+
+from odxtools.dataobjectproperty import DopBase
+from .globals import logger
+
+
+@dataclass
+class Case:
+    """This class represents a CASE."""
```

Comment:
this represents a multiplexer case, right? If so, please rename the class to `MultiplexerCase` and extend the docstring...